app/views/forms/uploads.html.erb

```diff
@@ -2,16 +2,29 @@
 
 <div class="row">
   <div class="col-md-8 col-sm-7">
-    <h2>Upload Grant Documents</h2>
-    <%= form_for(@upload, multipart: true) do |f| %>
-      <%= hidden_field(:upload, :user_id) %>
-      <%= hidden_field(:upload, :user_type) %>
-      <div class="field">
-        <%= f.file_field :file %>
+    <%= form_for(@upload, html: { class: 'new-upload form-horizontal', multipart: true }) do |f| %>
+      <div class="row">
+        <div class="col-xs-12">
+          <h2>Supportive Documentation</h2>
+          <hr />
+        </div>
       </div>
-      <div class="actions">
-        <%= f.submit 'Upload Document' %>
-        <%= link_to 'Nevermind', class: 'button' %>
+      <div class="row">
```

Comment:
I don't think you need this extra `.row` div, as form-group is effectively the same.